snap/squashfs/squashfs.go

```diff
@@ -202,7 +202,7 @@ func (s *Snap) Unpack(src, dstDir string) error {
 	usw := newUnsquashfsStderrWriter()
 
 	var output bytes.Buffer
-	cmd := exec.Command("unsquashfs", "-n", "-f", "-d", dstDir, s.path, src)
+	cmd := exec.Command("unsquashfs", "-no-xattrs", "-n", "-f", "-d", dstDir, s.path, src)
```

Comment:
this is fine. Maybe worth adding a doc comment to the function that the content is unpacked omitting any extended attributes, so capability bits, selinux labels and similar are all silently dropped.